.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1
```

Comment:
please for convenience always include a new line at the end of files

Reply:
weird it isn't showing up

![image](https://user-images.githubusercontent.com/27079992/135498525-c890bced-7786-48aa-bd4d-618af4972e9e.png)